ui/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/tabs-metacard.view.js

```diff
@@ -126,4 +127,14 @@ module.exports = TabsView.extend({
       this.$el.toggleClass('lacks-preview', !result.hasPreview())
     }
   },
+  determineDisabledContent: function() {
+    this.$el.toggleClass(
+      'is-editing-disabled',
+      properties.isEditingRestricted()
+    )
+    this.$el.toggleClass(
```

Comment:
We should use a different class here since there are two different ways for this class to be applied as of now (no preview or previews disabled).

If we still want to piggyback off the other name, we should move this check into the class toggle above so it's only done in a single place.